src/iojunk.jl

```diff
@@ -51,7 +51,7 @@ end
 
 # Read and return a byte from f. Throws EOFError if there is no more byte to read.
 function read(f::ReadableFile, ::Type{UInt8})
-	# This function needs to be fast because readbytes, readall, etc.
+	# This function needs to be fast because readbytes, readstring, etc.
```

Comment:
could re-write this to use a `Ref{UInt8}` object now. see usages in Base, for instance.